static/app/components/replays/keyMetrics.tsx

```diff
@@ -1,4 +1,3 @@
-import {Fragment} from 'react';
```

Comment:
I've noticed while working on this project it seems like in files where nothing is actually being imported from the react package that it will still include `import React from 'react;` - I know some envs require React to be imported in a file no matter what for the JSX or something like that so I'm assuming it has something to do with that.